avm/res/network/vpn-gateway/main.bicep

```diff
@@ -16,7 +16,7 @@ param natRules natRuleType[] = []
 @description('Required. The resource ID of a virtual Hub to connect to. Note: The virtual Hub and Gateway must be deployed into the same location.')
 param virtualHubResourceId string
 
-@description('Optional. BGP settings details.')
+@description('Optional. BGP settings details. You can specify either bgpPeeringAddress (for custom IPs outside APIPA ranges) OR bgpPeeringAddresses (for APIPA ranges 169.254.21.*/169.254.22.*), but not both simultaneously.')
```

Comment:
The change must also be added to the changelog.md for the next patch version `0.2.1`